### PR TITLE
move all packages from budgie & budgiePlugins scopes

### PR DESCRIPTION
**Pull request by getchoo**
## Description of changes

See https://github.com/NixOS/nixpkgs/pull/326366 for context. Based off the work in https://github.com/NixOS/nixpkgs/pull/319659

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
  </ul>
</details>
<details>
  <summary>21 packages built:</summary>
  <ul>
    <li>budgie-analogue-clock-applet</li>
    <li>budgie-backgrounds</li>
    <li>budgie-control-center</li>
    <li>budgie-control-center.debug</li>
    <li>budgie-desktop</li>
    <li>budgie-desktop-view</li>
    <li>budgie-desktop-with-plugins</li>
    <li>budgie-desktop.dev</li>
    <li>budgie-desktop.man</li>
    <li>budgie-gsettings-overrides</li>
    <li>budgie-media-player-applet</li>
    <li>budgie-screensaver</li>
    <li>budgie-screensaver.man</li>
    <li>budgie-session</li>
    <li>budgie-session.debug</li>
    <li>budgie-session.man</li>
    <li>budgie-user-indicator-redux</li>
    <li>magpie</li>
    <li>magpie.debug</li>
    <li>magpie.dev</li>
    <li>magpie.devdoc</li>
  </ul>
</details>

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by getchoo** [comment]
@ofborg test budgie

File: pkgs/by-name/bu/budgie-session/package.nix [hunk 1]
```diff
@@ -38,12 +40,17 @@ stdenv.mkDerivation (finalAttrs: {
...
+  ];
+
   patches = [
     (substituteAll {
       src = ./fix-paths.patch;
-      gsettings = "${glib.bin}/bin/gsettings";
-      dbusLaunch = "${dbus.lib}/bin/dbus-launch";
-      bash = "${bash}/bin/bash";
+      gsettings = lib.getExe' glib "gsettings";
+      dbusLaunch = lib.getExe' dbus "dbus-launch";
```
Comment (bobby285271) [inline pkgs/by-name/bu/budgie-session/package.nix:52]: (Not a request change) For future reference

https://github.com/NixOS/nixpkgs/blob/452a2137e0e33173e87e2bba01af8394ca8f68b1/pkgs/development/libraries/dbus/default.nix#L108-L109

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -37171,9 +37171,6 @@ with pkgs;
     appls = [ prio ];
   };
 
-  budgie = recurseIntoAttrs (callPackage ../desktops/budgie { });
-  budgiePlugins = recurseIntoAttrs (callPackage ../desktops/budgie/plugins { });
-
```
Comment (bobby285271) [inline pkgs/top-level/all-packages.nix:37176]: Maybe we should add throw message in aliases.nix?

**Comment by bobby285271** [comment]
@ofborg build budgie-desktop.passthru.tests magpie.passthru.tests budgie-screensaver.passthru.tests budgie-control-center.passthru.tests

**Comment by getchoo** [comment]
@ofborg build budgie-desktop.passthru.tests magpie.passthru.tests budgie-screensaver.passthru.tests budgie-control-center.passthru.tests

**Comment by bobby285271** [comment]
Welcome to the budgie maintainers team :-)

**Review by jtojnar — COMMENTED** [review]
:heavy_check_mark: Nice job.

**Pull request merged**